packages/web/src/components/edit/fields/RemixSettingsField/TrackInfo.tsx

```diff
@@ -19,28 +17,25 @@ type TrackInfoProps = {
 
 export const TrackInfo = (props: TrackInfoProps) => {
   const { trackId } = props
-  const image = useTrackCoverArt({
-    trackId,
-    size: SquareSizes.SIZE_150_BY_150
-  })
 
-  const { data: track } = useGetTrackById({ id: trackId })
+  const { data: track } = useTrack(trackId, {
+    select: (track) => pick(track, ['title', 'owner_id'])
+  })
 
   if (!track) return null
 
-  const { user } = track
-
   return (
     <SelectedValue>
-      <DynamicImage wrapperClassName={styles.artwork} image={image} />
+      <TrackArtwork
+        trackId={trackId}
+        size={SquareSizes.SIZE_150_BY_150}
+        mr='xs'
+        css={{ minHeight: 24, minWidth: 24 }}
```

Comment:
nit: could use spacing vars

Reply:
so i tried h/w, but they are overridden by their parent, so i just went with minHeight/minWidth to avoid changing the parent and potentially introducing bugs with all the other components that have the same parent